packages/std/src/query/wasm.rs

```diff
@@ -21,4 +21,21 @@ pub enum WasmQuery {
         /// Key is the raw key used in the contracts Storage
         key: Binary,
     },
+    /// returns a ContractInfoResponse with metadata on the contract from the runtime
+    ContractInfo { contract_addr: String },
+}
+
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
+pub struct ContractInfoResponse {
+    pub code_id: u64,
+    /// address that instantiated this contract
+    pub creator: String,
+    /// admin who can run migrations (if any)
+    pub admin: Option<String>,
+    /// if set, the contract is pinned to the cache, and thus uses less gas when called
+    pub pinned: bool,
+    /// block height this was created at
+    pub created_at: u64,
```

Comment:
Do we need this info? I'm a bit unsure what the best format is. We have AbsoluteTxPosition, block height or block time. The client does not get it.

If we don't need this, let's consider leaving that out to be decided later on.

Reply:
we expose AbsoluteTxPosition to the client, which is a superset of block height. 

I'm fine to remove this field. I cannot see a real need for it

Reply:
Ah, right. It became available with the protobuf upgrade. But then it is stripped off when using the higher level CosmWasm client in CosmJS.

I would favour to remove it for now in order to allow is thinking about the format carefully.